### DISCUSSION TITLE
LinuxFr article on VulkanSceneGraph

### DISCUSSION THREAD

**Discussion by robertosfield** [vsg-dev/VulkanSceneGraph#675] [General]
Just came across this article about the VulkanSceneGraph on linuxfr.org. The article provides a nice high level intro to the project. 

The original can be read at:
https://linuxfr.org/news/vulkanscenegraph-un-graphe-de-scene-en-c

I was able to translate this to English:
https://linuxfr-org.translate.goog/news/vulkansce

The author geos by the name small_dukc on LinuxFr but I don't know anything more, perhaps they are here active here as well but go by a different name.  Either way thanks to them for writing this article.

**Comment by LaurensVoerman** [comment]
The google translate link got truncated, let's try again:
https://linuxfr-org.translate.goog/news/vulkanscenegraph-un-graphe-de-scene-en-c?_x_tr_sl=fr&_x_tr_tl=en
Laurens.
